docs/contributing/writing-instrumentation.md

```diff
@@ -68,6 +68,47 @@ include("instrumentation:yarpc-1.0:library")
 include("instrumentation:yarpc-1.0:testing")
 ```
 
+## Instrumentation Submodules
+
+When writing instrumentation that requires submodules for different versions, the name of each
```

Comment:
How about adding related content in `Folder Structure` part directly? It might look more concentrated.

Reply:
good idea, i've made the `instrumentation submodules` section an H3 so it's a subsection of `Folder Structure`